concepticondata/references/references.bib

```diff
@@ -1,3 +1,13 @@
+@Article{Huttenlocher1979,
+  author    = {Huttenlocher, Janellen and Lui, Felicia},
+  title     = {The semantic organization of simple nouns and verbs},
+  number    = {1},
```

Comment:
The exported bibtex citation indicates that the issue is 2, not 1. Could it be the case here?

Reply:
Thanks, yes, this can be the case.